### ISSUE TITLE
TODO list

### ISSUE THREAD

**Issue by z33ky** [Palm-Studios/sh3redux#77]
For people interested in contributing we should have a list of TODO items.

We should have some more abstract, future goals so we know what we're working towards, as well as more concrete, easier to reach ones so people know what they can work on *now*.

We should open issues and assign ourself for things that we are currently working on and leave a note in the TODO list to check the issues first, so that there's no wasted effort. A label ("WIP"?) would be useful to distinguish these tickets from other issues.

**Comment by Quaker762** [comment]
Yeah sure thing, that sounds good. 

I'm probably best at working on some of the formats, as my game logic programming is pretty awful (my statemanager design I usually use is VERY unsafe haha). Would we put the TODO list in `Readme.MD`??

I'll create a WIP label now. I want to get these PR's I have open out of the way so I can clean up some of my branches as it's a bit chaotic at the moment.

EDIT: Hahahaha

**Comment by Quaker762** [comment]
Oh, and in case you didn't know, there's a site for us I made [here](https://palm-studios.github.io/sh3-redux/) (home page is one up).

I've made it so every time or every other time I update my master repo folder, it automatically updates the documentation so it's current.

**Comment by z33ky** [comment]
> Would we put the TODO list in `Readme.MD`?

We should perhaps roughly describe the state of development, but otherwise I'd keep it a separate file. It's not uncommon to have a top-level file named `TODO` or the likes.

> Oh, and in case you didn't know, there's a site for us I made [here](https://palm-studios.github.io/sh3-redux/) (home page is one up).

Neat. I'll check it out.

**Comment by Quaker762** [comment]
Sure thing, did you want to make the file? You seem much more competent at design and splitting things up than I am haha. If it was up to me it'd just be a mush of file types and systems.

**Comment by z33ky** [comment]
Sure. I'll give it a go when I'm home (in a few hours..).

**Comment by Quaker762** [comment]
> Sure. I'll give it a go when I'm home (in a few hours..).

That reminds me, what timezone are you in? It'd be good to sync this stuff up a bit so we can have 110% efficiency haha. I'm in Melbourne Australia, so AEST.

**Comment by z33ky** [comment]
I live in Germany, so CET, a.k.a. UTC+1 (unless it's summer and some... "people"... think it's awesome to use UTC+2 instead).

**Comment by z33ky** [comment]
Actually it might be nicer to just refer to the issue tracker and make all TODO items an issue. This allows for easier discussion and people can subscribe to specific items.
Checkout https://github.com/z33ky/sh3redux/blob/todo/TODO for a list I made from the top of my head.

**Comment by Quaker762** [comment]
> Actually it might be nicer to just refer to the issue tracker and make all TODO items an issue. This allows for easier discussion and people can subscribe to specific items.

Yeah that sounds a little bit easier to follow. For now I'll put the TODO you've made on the Wiki.
P.S SILENT HILL has no overlay whatsoever, which makes our lives a LOT easier. The hardest thing to do imho will be the camera. Whoever programmed that at Konami is pretty good haha.
